src/check_result.jl

```diff
@@ -0,0 +1,51 @@
+# For once you have the sensitivity by two methods (e.g  both finite-differencing and  AD)
```

Comment:
Basically this file replaces `isapprox.jl`
though the removal of `isapprox.jl` is not done yet

Reply:
+1 for more human-readable error messages!